auth0/src/test/java/com/auth0/android/MockAuth0.java

```diff
@@ -0,0 +1,35 @@
+package com.auth0.android;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.squareup.okhttp.HttpUrl;
+
+/**
+ * Mock implementation of {@linkplain Auth0} for tests.
+ */
```

Comment:
Add a javadoc line saying that this is meant to remove the "https" enforcement/check.

Reply:
Added docs in new commit